main.cpp

```diff
@@ -1,162 +1,241 @@
+{ 
+    ignoreUnused(rentalDuration, carType); //passing each function parameter to the ignoreUnused() function
+    return {}; //if your function returns something other than void, add 'return {};' at the end of it.
+} 
+
+/*
+ 1)
+ */
+void soundOsc (int waveType = 1, double amplitude = 0.1) 
+{
+    ignoreUnused(waveType, amplitude);
+
+}
+/*
+ 2)
+ */
+double sumLR (double leftCh = 0, double rightCh = 0)
+{
+    ignoreUnused(leftCh, rightCh);
+    return{};
+}
+
+/*
+ 3)
+ */
+int waveType (int waveSelSw = 0) 
+{
+    ignoreUnused(waveSelSw);
+    return{};
+}
+
+/*
+ 4)
+ */
+double amplitudeEnv (double ampEnv = 0.1, double oscMixerOut = 0)
+{
+    ignoreUnused(ampEnv, oscMixerOut);
+    return{};
+}
+
+/*
+ 5)
+ */
+bool selectParam (bool switchOne = 0, bool switchTwo = 0)
+{
+    ignoreUnused(switchOne, switchTwo);
+    return{};
+} 
+
+/*
+ 6)
+ */
+float displaySelectedOscType (bool switchOne = 0, int encoderSel = 0)
+{
+    ignoreUnused(switchOne, encoderSel);
+    return{};
+}
+
+/*
+ 7)
+ */
+bool isSomethingConnected (bool inputOneSwitch = 0, bool inputTwoSwitch = 0)
+{
+    ignoreUnused(inputOneSwitch, inputTwoSwitch);
+    return{};
+}
+
+/*
+ 8)
+ */
+int checkBatteryStatus (int batteryTemp = 0, int batteryCharge = 0, bool batteryCharging = 0)
+{
+    ignoreUnused(batteryTemp, batteryCharge, batteryCharging);
+    return{};
+}
+
+/*
+ 9)
+ */
+int detectInputType (float inputImpedance = 0, double inputAmplitue = 0.1)
+{
+    ignoreUnused(inputImpedance, inputAmplitue);
+    return {};
+}
+
+/*
+ 10)
+ */
+int activateArp (bool isMidiInconnected = 0, int arpSelType = 0)
+{
+    ignoreUnused(isMidiInconnected, arpSelType);
+    return {};
+}
+
+/*
+ MAKE SURE YOU ARE NOT ON THE MASTER BRANCH
+
+ Commit your changes by clicking on the Source Control panel on the left, entering a message, and click [Commit and push].
+ 
+ If you didn't already: 
+    Make a pull request after you make your first commit
+    pin the pull request link and this repl.it link to our DM thread in a single message.
+
+ send me a DM to review your pull request when the project is ready for review.
+
+ Wait for my code review.
+ */
+
+int main()
+{
+    //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
+    auto carRented = rentACar(6, 2); 
+    
+    //1)
+    soundOsc(1,0);
+
+    //2)
+    auto sumChannels = sumLR(0,0);
+
+    //3)
+    auto wave = waveType(0);
+
+    //4)
+    auto envelopeOut = amplitudeEnv(0,0);
+
+    //5)
+    auto userSelection = selectParam(2,0);
+    
+    //6)
+    auto userDisplay = displaySelectedOscType(0,0);
+
+    //7)
+    auto connectionStatus = isSomethingConnected(0,0);
+
+    //8)
+    auto batteryInfo = checkBatteryStatus(0,0,1);
+    
+    //9)
+    auto inputPreAmp = detectInputType(0,0);
+    
+    //10)
+    auto arp = activateArp(0,0);
```

Comment:
1) add space after `,` to make your function arguments easier to read.
2) Booleans need to be initialized with `true` or `false`. 
never initialize booleans with integers
- [ ] revise